.github/workflows/check.yml

```diff
@@ -123,6 +123,6 @@ jobs:
     - uses: actions/setup-python@v4
       with:
         python-version: "3.10"
-    - run: python3 scripts/bindcheck.py porcupine
-    - run: python3 scripts/bindcheck.py more_plugins
-    - run: python3 scripts/bindcheck.py tests
+        cache: pip
+    - run: pip install flake8-tkinter
```

Comment:
I don't want things to break if you add newer, better warnings :) This should be `flake8-tkinter==0.5.0`, either here or in `requirements-dev.txt`.